src/patternfly/components/Popover/popover.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="pf-c-popover {{pf-c-popover-modifier}}">
+  <div class="pf-c-popover__container">
+    <div class="pf-c-popover__arrow"></div>
+    <div class="pf-c-popover__content">
+      <div class="pf-c-popover__header">
+        <div class="pf-c-popover__header-text">{{header}}</div>
+        <div class="pf-c-popover__close">
+          <button class="pf-c-button pf-m-action" aria-label="Remove">
+            <i class="fas fa-times"></i>
+          </button>
+        </div>
+      </div>
+      <div class="pf-c-popover__body">
+        {{> @partial-block}}
+      </div>
+    </div>
+  </div>
+  {{!-- {{> popoverParent }} --}}
```

Comment:
Finally figured this out over here - popoverParent should be kebab-cased and needs to be defined slightly differently in the base component:

  {{#> popover-parent }}
  {{/popover-parent}}

this is my entire component from the code we shared before:

```
<div class="pf-c-tooltip {{pf-c-tooltip-modifier}}">
  <div class="pf-c-tooltip__container">
    <div class="pf-c-tooltip__arrow"></div>
    <div class="pf-c-tooltip__content">
      {{> @partial-block }}
    </div>
  </div>
  {{#> tooltip-parent }}
  {{/tooltip-parent}}
</div>
```